_aggregations/metric/geobounds.md

```diff
@@ -26,30 +37,55 @@ GET opensearch_dashboards_sample_data_ecommerce/_search
   }
 }
 ```
+{% include copy-curl.html %}
+
+## Example response
 
-#### Example response
+The response is as follows:
 
 ```json
-"aggregations" : {
-  "geo" : {
-    "bounds" : {
-      "top_left" : {
-        "lat" : 52.49999997206032,
-        "lon" : -118.20000001229346
-      },
-      "bottom_right" : {
-        "lat" : 4.599999985657632,
-        "lon" : 55.299999956041574
+{
+  "took": 16,
+  "timed_out": false,
+  "_shards": {
+    "total": 1,
+    "successful": 1,
+    "skipped": 0,
+    "failed": 0
+  },
+  "hits": {
+    "total": {
+      "value": 4675,
+      "relation": "eq"
+    },
+    "max_score": null,
+    "hits": []
+  },
+  "aggregations": {
+    "geo": {
+      "bounds": {
+        "top_left": {
+          "lat": 52.49999997206032,
+          "lon": -118.20000001229346
+        },
+        "bottom_right": {
+          "lat": 4.599999985657632,
+          "lon": 55.299999956041574
+        }
       }
     }
   }
- }
 }
 ```
 
 ## Aggregating geoshapes
 
-To run an aggregation on a geoshape field, first create an index and map the `location` field as a `geo_shape`:
+You can run a `geo_bounds` aggregation on geoshapes.
+
+Prepare an example by inserting an index containing a geoshape field:
```

Comment:
Please split the example into 2 code blocks:

1. Prepare an example by creating an index containing a geoshape field
2. Ingest documents into the index
```

Reply:
```suggestion
Prepare an example by inserting an index containing a geoshape field. GeoJSON input specifies longitude first:
```